### PR TITLE
Added the UI Feature Update in the Password Visibility Toggle

### PR DESCRIPTION
**Pull request by VarunT11**
**Issue Reference** : https://github.com/odk-x/tool-suite-X/issues/200

**Small Description of what this PR does** : 
- Adds the Android Support Design Dependency in the **_build.gradle_** File of the App Module of the Project.
- Removes the _Checkbox_ from the Layout XML File of the **_login_fragment.xml_** and **_password_dialog_layout.xml_** Files and changes the normal _EditText_ to _TextInputEditText_ provided by the Android Support Dependency.
- Removed the use of the _Checkbox_ from the _**LoginFragment.java**_ and _**PasswordDialogFragment.java**_ file.

**Screenshots** :
Screenshot_20210313-122355
Screenshot_20210313-122412
Screenshot_20210313-124715
Screenshot_20210313-124701

### CODE REVIEW

**Comment by wbrunette** [comment]
runtests

File: services_app/build.gradle [hunk 1]
```diff
@@ -122,6 +122,7 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'androidx.preference:preference:1.1.1'
     implementation 'androidx.fragment:fragment:1.3.0-beta01'
+    implementation 'com.android.support:design:28.0.0'
```
Comment (linl33) [inline services_app/build.gradle:125]: Please update this library, see https://material.io/develop/android/docs/getting-started#new-namespace-and-androidx

**Review by linl33 — CHANGES_REQUESTED** [review]
@VarunT11 Thanks for implementing this feature!

The new password field works great, just a few comment
1. Please remove the now unused `import`s in the fragment classes
2. Could you bring the changes to `password_challenge_dialog_layout` as well? That layout is used to enter the admin password.

**Comment by VarunT11** [comment]
**Small Description of Changes Performed**:
- Added the Password Visibility Toggle Feature to the _**AdminPasswordChallengeFragment.**_
- Removed the Unused Imports from _**LoginFragment**_ , _**PasswordDialogFragment**_ and _**AdminPasswordChallengeFragment.**_
- Updated the _**build.gradle**_ File by updating the Libraries.

**Screenshots**:
Screenshot_20210315-124801__01
Screenshot_20210315-124806__01

**PS**: @linl33 The Requested Changes have been performed. Please review them!

**Comment by linl33** [comment]
runtests

**Review by linl33 — APPROVED** [review]

**Review by wbrunette — APPROVED** [review]

**Pull request merged**